bin/kvstore_v2/src/main.rs

```diff
@@ -0,0 +1,176 @@
+use api::{check_bootstrap_config, consensus_api::ConsensusEngine, NodeConfig};
+use api_types::{default_recover::DefaultRecovery, ConsensusApi, ExecutionChannel, ExecutionLayer};
+use clap::Parser;
+use cli::Cli;
+use execution_channel::ExecutionChannelImpl;
+use flexi_logger::{detailed_format, FileSpec, Logger, WriteMode};
+use pipeline_blockchain::*;
+use server::ServerApp;
+use std::{error::Error, sync::Arc, thread};
+
+struct TestConsensusLayer {
+    consensus_engine: Arc<dyn ConsensusApi>,
+}
+
+impl TestConsensusLayer {
+    fn new(node_config: NodeConfig, execution_client: Arc<dyn ExecutionChannel>) -> Self {
+        let execution_layer = ExecutionLayer {
+            execution_api: execution_client,
+            recovery_api: Arc::new(DefaultRecovery {}),
+        };
+        Self { consensus_engine: ConsensusEngine::init(node_config, execution_layer, 1337) }
+    }
+
+    async fn run(self) {
+        loop {
+            tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
+        }
+    }
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
```

Comment:
Add comments of known limitations here, i.e. TODOs for using as an example of production.